docs/content/en/tasks/backup-restore/_index.md

```diff
@@ -115,32 +115,27 @@ To perform a backup of a Cassandra datacenter, create the following custom resou
 
 ```yaml
 apiVersion: medusa.k8ssandra.io/v1alpha1
-kind: CassandraBackup
+kind: MedusaBackupJob
 metadata:
   name: medusa-backup1
 spec:
   cassandraDatacenter: dc1
-  name: medusa-backup1
 ```
 
```

Comment:
The docs says in a few places 

>The definition above requires a `medusa-bucket-key` to be created...

I realize that isn't part of this PR, but we can change to

>The definition above requires a secret named `medusa-bucket-key` to be created...